streams/README.md

```diff
@@ -21,32 +21,35 @@ Arrays.stream(persons)
     java Streams 100000 200
     ```
     ```
```

Comment:
Can you please rename this demo to:

"Java Streams API Benchmark on GraalVM" instead of "Abstraction Without Regret When Running on GraalVM"

Reply:
Will do.